tests/acceptance/SubscriberManageImportExportCest.php

```diff
@@ -101,6 +101,21 @@ private function chooseListAndConfirm(\AcceptanceTester $i) {
     $i->waitForText('Import again');
   }
 
+  private function createNewListAndConfirm(\AcceptanceTester $i) {
+    $newListName = 'Simple List';
+    // first create a new list
+    $i->click('a.mailpoet_create_segment');
+    $i->fillField('input#new_segment_name', $newListName);
+    $i->fillField('textarea#new_segment_description', 'This is just a simple list.');
+    $i->click('input#new_segment_process');
+    // trigger dropdown to display selections and search for recently created list
+    $i->waitForElementVisible('input.select2-search__field');
```

Comment:
We have a helper function for selecting in select2. I think you can use `$i->selectOptionInSelect2($newListName);`
See https://github.com/mailpoet/mailpoet/blob/master/tests/_support/AcceptanceTester.php#L122